pipeline-modules/app-service/azure/node-build.yaml

```diff
@@ -0,0 +1,65 @@
+provisioner: azure
+name: node-build
+version: 1
+revision: 1
+displayName: Node App Build
+description: Build source code using Node
+target: ""
+category: build
+keywords:
+  - Node
+  - Install
+  - Build
+author: CloudCover
+meta: {}
+inputs:
+
+  properties:
+    package_manager:
+      title: Package manager
+      description: "Your package manager: npm, pnpm, yarn or node"
+      type: string
+      default: npm
+      enum:
+        - npm
+        - yarn
+        - pnpm
+        - node
+    npm_build_args:
+      title: Package manager arguments
+      description: Arguments for your Package manager command
+      type: array
+      default: ["install"]
+      items:
+        type: string
+    working_dir:
+      title: Working directory
+      description: The current working directory to execute command
+      type: string
+      default: repo
+    node_js_version:
+      title: Node.js Version
+      description: The version of Node.js like 18.x
+      type: string
+      default: '18.x'  
+    sha_tag:
+      title: SHA Git commit tag
+      description: Resolved Commit hash for Input git artifact
+      type: string           
+  internal:
+    - working_dir
+    - sha_tag       
+template: |
+  steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: {{node_js_version}}
+    displayName: "Install Node.js with version {{node_js_version}}"
+
+  - script: |
+      echo "Start install & building node app......"
+    {% for arg in npm_build_args %}
+      {{ package_manager }} {{ arg }}
+    {% endfor %}      
+    workingDirectory: {{ working_dir }}
+    displayName: 'npm install and build'
```

Comment:
The diplayname should be Node Build or something like "App Build"

Reply:
Update with Node App build